.circleci/config.yml

```diff
@@ -6,98 +6,57 @@ defaults:
 jobs:
   build:
     docker:
-      - image: circleci/python:2.7
+      - image: circleci/python:3.8
```

Comment:
so for cicle ci lets move few things in github actions.

lets add the code checks:
- lint
- build
- test
- coverage